app/views/product_drives/index.html.erb

```diff
@@ -103,6 +107,7 @@
                   <td><%= product_drive.name %></td>
                   <td><%= product_drive.start_date.strftime("%m-%d-%Y") %></td>
                   <td><%= product_drive.end_date&.strftime("%m-%d-%Y") %></td>
+                  <td><%= product_drive.tags.map(&:name).sort.join(", ") %></td>
```

Comment:
This feels like a method that should live on the Taggable concern.

Reply:
True. Fixed in 99f4f973392141e81549109f24ef514b4ee3fddb.